ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/BlsToExecutionOperationPoolTest.java

```diff
@@ -140,6 +141,23 @@ void shouldAddMaxItemsToPool() {
     assertThat(pool.getItemsForBlock(state)).contains(item);
   }
 
+  @Test
+  void shouldSelectLocalMessagesBeforeRemoteMessage() {
```

Comment:
NIT: what is selected is the operation, not the message :)

`s/shouldSelectLocalMessagesBeforeRemoteMessage/shouldSelectLocalOperationBeforeRemoteOperation`

Reply:
Or even `shouldSelectLocalItemsBeforeRemoteItems` if we want to be consistent with other methods 🤷